firebase_admin/db.py

```diff
@@ -69,6 +78,49 @@ def _parse_path(path):
     return [seg for seg in path.split('/') if seg]
 
 
+class Stream(object):
+    """Class that handles the streaming of data node changes from server"""
+    def __init__(self, url, build_headers, stream_handler, stream_id):
+        """Initialize the streaming object"""
+        self.url = url
+        self.build_headers = build_headers
+        self.stream_handler = stream_handler
+        self.stream_id = stream_id
+        self.sse = None
+        self.thread = None
+        self.start()
+
+    def start(self):
+        """Start the streaming by spawning a thread"""
+        self.thread = threading.Thread(target=self.start_stream)
+        self.thread.start()
+        return self
+
+    def start_stream(self):
+        """Streaming function for the spawned thread to run"""
+        self.sse = SSEClient(
```

Comment:
Can you move this to the start() method above, before the thread is started? Then I think you will be able to drop the sleep() call in close().